firebase-perf/src/test/java/com/google/firebase/perf/application/FragmentStateMonitorTest.java

```diff
@@ -243,6 +244,21 @@ public void fragmentTraceCreation_dropsTrace_whenFragmentNameTooLong() {
     verify(mockTransportManager, times(0)).log(any(TraceMetric.class), any());
   }
 
+  @Test
+  public void fragmentTraceCreation_addsSpecialAttributeValue_whenFragmentDoesNotHaveParent() {
```

Comment:
Curious - Do we have tests for custom attributes when there is a parent fragment available?

Reply:
Good question, I don't believe we do